scripts/enable-ecs-agent-gpu-support.sh

```diff
@@ -22,9 +23,74 @@ enabled=1
 exclude=libglvnd-*
 EOF
 
-# this repo is temporary and only used for installing the system-release-nvidia package
 sudo mv $tmpfile /etc/yum.repos.d/amzn2-nvidia-tmp.repo
-# system-release-nvidia creates an nvidia repo file at /etc/yum.repos.d/amzn2-nvidia.repo
+
+# only install open driver for post-kepler gpus
+if [[ $AMI_TYPE != "al2keplergpu" ]]; then
+    sudo yum install -y yum-plugin-versionlock \
```

Comment:
why do we need versionlock? I dont see it being used below

Reply:
see the proposed userdata in the description:
```
#!/bin/bash
/var/lib/ecs/scripts/install-nvidia-open-kmod.sh
yum versionlock \
cuda-* \
cuda-drivers-* \
cuda-drivers-fabricmanager-* \
nvidia-container-toolkit-*
```